source/codegen/metadata/nifgen/enums.py

```diff
@@ -1557,61 +1150,65 @@
     'Waveform': {
         'values': [
             {
-                'documentation': {
-                    'description': 'Sinusoid waveform'
-                },
                 'name': 'NIFGEN_VAL_WFM_SINE',
                 'value': 1
             },
             {
-                'documentation': {
-                    'description': 'Square waveform'
-                },
                 'name': 'NIFGEN_VAL_WFM_SQUARE',
                 'value': 2
             },
             {
-                'documentation': {
-                    'description': 'Triange waveform'
-                },
                 'name': 'NIFGEN_VAL_WFM_TRIANGLE',
                 'value': 3
             },
             {
-                'documentation': {
-                    'description': 'Positive ramp waveform'
-                },
                 'name': 'NIFGEN_VAL_WFM_RAMP_UP',
                 'value': 4
             },
             {
-                'documentation': {
-                    'description': 'Negative ramp waveform'
-                },
                 'name': 'NIFGEN_VAL_WFM_RAMP_DOWN',
                 'value': 5
             },
             {
-                'documentation': {
-                    'description': 'Constant voltage'
-                },
                 'name': 'NIFGEN_VAL_WFM_DC',
                 'value': 6
             },
             {
-                'documentation': {
-                    'description': 'White noise'
-                },
                 'name': 'NIFGEN_VAL_WFM_NOISE',
                 'value': 101
             },
             {
-                'documentation': {
-                    'description': 'User-defined waveform as defined by the niFgen_DefineUserStandardWaveform function.'
-                },
                 'name': 'NIFGEN_VAL_WFM_USER',
                 'value': 102
             }
         ]
+    },
+    'CalAdcInput': {
```

Comment:
We dont need to add this since the attribute using these values `NIFGEN_ATTR_CAL_ADC_INPUT` is not supported

Reply:
removed the enum